sdk/core/ci.yml

```diff
@@ -31,7 +31,9 @@ stages:
   - template: ../../eng/pipelines/templates/stages/archetype-sdk-client.yml
     parameters:
       ServiceDirectory: core
+      # CI has static code analisys disabled, while LiveTest will have it enabled
       CtestRegex: azure-core.
+      LiveTestCtestRegex: azure-core.
```

Comment:
What's the purpose of this parameter?

Reply:
Enables liveTesting stage for a pipeline.

When it is not set, there are not liveTests run for that pipeline.  Then the value defines the regex that is used to run the tests like  `ctest -V -R azure-core.`

All the tests from azure-core are prefixed with `azure-core.` , so this regex would only run those tests.

For Storage, for instance, the regex is azure-storage. :)